app/js/controllers.js

```diff
@@ -4515,6 +4519,17 @@ angular.module('myApp.controllers', ['myApp.i18n'])
         }
         $rootScope.$broadcast('settings_changed')
       }
+
+      $scope.toggleNightmode = function () {
+        $scope.appearance.nightmode = !$scope.appearance.nightmode;
+        if (!$scope.appearance.nightmode) {
+          Storage.remove('appearance_nightmode');
```

Comment:
Perhaps we can store this key in a config somewhere and reference it everywhere else.

Reply:
I don't know how to do that and I don't want to get too much into the code base. I propose, if you don't mind and if the code is good enough, to make this change later on master.